frontend/src/Components/Charts/TrafficStops/TrafficStops.js

```diff
@@ -493,6 +494,7 @@ function TrafficStops(props) {
               onChange={handleStopPurposeSelect}
               options={[PURPOSE_DEFAULT].concat(stopTypes)}
             />
+            <p style={{ marginTop: '10px' }}>{displayDefinition(purpose)}</p>
```

Comment:
Let's only show this p tag if the purpose is not set to `PURPOSE_DEFAULT` so the extra space isn't added.